lectia-2.js

```diff
@@ -1,3 +1,14 @@
-// https://miorita.scoalaweb.ro/lessons/2-if
-// Scrie un program care să o ajute pe Miorița să ajungă la oricare din pășuni
-// Programul trebuie să funcționeze și cu gard, și fără gard
+reset()
+if ( canMove() ) {
+        move()
+        move()
+        move()
+}
+else {
```

Comment:
Să te gândești dacă e vreo metodă să faci fără `else`

Reply:
Aici ar trebui sa fac cu variabila, cum ai facut si tu in exemplu si sa adaug undeva(nu stiu exact unde ar trebui) if(!canMove()) turnRight.

Reply:
Am reusit, totusi sa fac fara else. Am lasat varianta mai jos, sub else.

Am lasat codul pe care l-ai scris tu acolo, si am adaugat in while if(!canMove()) si acum merge.